v2.0/admin-ui-access-and-navigate.md

```diff
@@ -107,6 +84,58 @@ Down Since | How long the node has been down.
 
 When you decommission a node, CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down. See [Remove Nodes](remove-nodes.html) for more information.
 
+### Node Map (Enterprise)
+
+The **Node Map** is an [enterprise-only](enterprise-licensing.html) feature that gives you a visual representation of the geographical configuration of your cluster. To set up and enable the Node Map, see [Enable Node Map](enable-node-map.html). The Node Map consists of the following components:
+
+**Node component**
+
+<img src="{{ 'images/v2.0/admin-ui-node-components.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
+
+**Region component**
```

Comment:
Should we list region before node, if that's the hierarchy from a user's perspective?

Reply:
The node has the individual metrics that are then aggregated to the region level. Hence the hierarchy in the doc.

Reply:
Yeah, but my question is about UX: Should we list these in the order that users will encounter them? They won't see nodes until they drill into regions, right? If so, I'd list regions first.